docs/upgrading-solidus/v3.4.mdx

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 1
+---
+
+import PRLink from '@site/src/theme/PRLink';
+import MinimalRequirements from '@site/src/theme/MinimalRequirements';
+
+# Solidus v3.4 (2023-04-TBD)
+
+<MinimalRequirements ruby="2.7" rails="6.0" />
+
+This is a slimmer release, but a very important one as it'll be the last on the 3.x series. [Following our policy](https://solidus.io/release_policy/#last_minor_before_major), there won't be other changes except removing deprecated code before we release v4. That means if you adapt your store to be free of warnings, you should be ready to go when v4 is out (and it's just around the corner!).
+
+Please, remember to review our generic [upgrade guides](/getting-started/upgrading-solidus.mdx). You can also check the complete [Changelog](https://github.com/solidusio/solidus/blob/v3.4/CHANGELOG.md) in our repository.
```

Comment:
I'm always afraid that people won't read it accurately and miss running the `solidus:update` task. What about adding it anyway as some form of TLDR? I'm sure they will go in the guide to better understand how to use the output of that task.

Reply:
I think you're right. I added the explicit task that needs to be run. Let me know what you think.